src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.scss

```diff
@@ -51,8 +52,24 @@
   }
 }
 
-.moreInformation {
+.moreInformationLink {
   padding-top: 16px;
+  color: $blue;
+  cursor: pointer;
+}
+
+.moreInformation {
+  grid-area: moreInformation;
+  display: grid;
+  grid-template-columns: 190px auto;
+  padding-top: 6px;
+}
+
+.moreInfoColumn {
+  width: 190px;
+  div {
+    padding-bottom: 6px;
```

Comment:
As said earlier, you may need a padding-left here as per XD

Reply:
i added a grid-colum-gap from the parent

Reply:
I still dont see the padding on the left as per the xd screenshot above, can you?

Reply:
where are you not seeing the padding, in the code or in the review URL? I can see it in the review URL:
![image](https://user-images.githubusercontent.com/6040010/125839372-6d0f5e3d-dab8-4f0f-908b-64b24abcb12c.png)

Reply:
This is now done @jyothishnt . can you please review/approve. Thanks